oper8/verify_resources.py

```diff
@@ -144,7 +144,7 @@ def verify_pod(object_state: dict) -> bool:
 def verify_job(object_state: dict) -> bool:
     """Verify that a job has completed successfully"""
     # https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/#JobStatus
-    return _verify_condition(object_state, "Completed", True)
+    return _verify_condition(object_state, "Complete", True)
```

Comment:
@pit1sIBM this is a NIT but should we replace this with a constant like we do for [Deployments](https://github.com/pit1sIBM/oper8/blob/5d226f458b43f3ffaae68b4a90d5d41d6f109763/oper8/verify_resources.py#L26)

Reply:
Done, thank you